docker-compose.yml

```diff
@@ -7,6 +7,12 @@ services:
       dockerfile: Dockerfile
     volumes:
       - yitam-mcp-dist:/app/dist
+    ports:
+      - "127.0.0.1:3030:3030"
+    environment:
+      - PORT=3030
+      - MCP_SERVER_HOST=0.0.0.0
+      - NODE_ENV=production
```

Comment:
where can we set the other ENVs for mcp server, we might need to use smth like below on docker compose?

```
    env_file:
      - prod.env
```